When building `5.1.2-1.0`, version `*-TDP-0.1.0-SNAPSHOT` is needed

When building phoenix in branch `5.1.2-TDP` I have this error:

```
[INFO] -----------< org.apache.phoenix:phoenix-hbase-compat-2.4.1 >------------
[INFO] Building Phoenix Hbase 2.4.1 compatibility 5.1.2-1.0              [2/14]
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for org.apache.hbase:hbase-common:jar:2.1.10-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hbase:hbase-hadoop-compat:jar:2.1.10-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hbase:hbase-hadoop2-compat:jar:2.1.10-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hbase:hbase-protocol-shaded:jar:2.1.10-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hbase:hbase-protocol:jar:2.1.10-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hadoop:hadoop-auth:jar:3.1.1-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hadoop:hadoop-common:jar:3.1.1-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hbase:hbase-metrics-api:jar:2.1.10-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hbase:hbase-client:jar:2.1.10-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hbase:hbase-zookeeper:jar:2.1.10-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hbase:hbase-metrics:jar:2.1.10-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hadoop:hadoop-hdfs:jar:3.1.1-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hadoop:hadoop-annotations:jar:3.1.1-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
Downloading from java.net.Releases: https://maven.java.net/content/repositories/releases/org/glassfish/javax.el/maven-metadata.xml
Downloading from jvnet-nexus-snapshots: https://maven.java.net/content/repositories/snapshots/org/glassfish/javax.el/maven-metadata.xml
[WARNING] Could not transfer metadata org.glassfish:javax.el/maven-metadata.xml from/to java.net.Releases (https://maven.java.net/content/repositories/releases): Access denied to: https://maven.java.net/content/repositories/releases/org/glassfish/javax.el/maven-metadata.xml , ReasonPhrase:Forbidden.
[WARNING] Could not transfer metadata org.glassfish:javax.el/maven-metadata.xml from/to jvnet-nexus-snapshots (https://maven.java.net/content/repositories/snapshots): Access denied to: https://maven.java.net/content/repositories/snapshots/org/glassfish/javax.el/maven-metadata.xml , ReasonPhrase:Forbidden.
[WARNING] The POM for org.apache.hadoop:hadoop-distcp:jar:3.1.1-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[WARNING] The POM for org.apache.hadoop:hadoop-mapreduce-client-core:jar:3.1.1-TDP-0.1.0-SNAPSHOT is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for Apache Phoenix 5.1.2-1.0:
[INFO] 
[INFO] Apache Phoenix ..................................... SUCCESS [  0.953 s]
[INFO] Phoenix Hbase 2.4.1 compatibility .................. FAILURE [  0.243 s]
[INFO] Phoenix Hbase 2.4.0 compatibility .................. SKIPPED
[INFO] Phoenix Hbase 2.3.0 compatibility .................. SKIPPED
[INFO] Phoenix Hbase 2.2.5 compatibility .................. SKIPPED
[INFO] Phoenix Hbase 2.1.6 compatibility .................. SKIPPED
[INFO] Phoenix Core ....................................... SKIPPED
[INFO] Phoenix - Pherf .................................... SKIPPED
[INFO] Phoenix - Tracing Web Application .................. SKIPPED
[INFO] Phoenix Client Parent .............................. SKIPPED
[INFO] Phoenix Client ..................................... SKIPPED
[INFO] Phoenix Client Embedded ............................ SKIPPED
[INFO] Phoenix Server JAR ................................. SKIPPED
[INFO] Phoenix Assembly ................................... SKIPPED
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  1.574 s
[INFO] Finished at: 2023-05-31T10:44:23Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project phoenix-hbase-compat-2.4.1: Could not resolve dependencies for project org.apache.phoenix:phoenix-hbase-compat-2.4.1:jar:5.1.2-1.0: The following artifacts could not be resolved: org.apache.hadoop:hadoop-auth:jar:3.1.1-TDP-0.1.0-SNAPSHOT, org.apache.hadoop:hadoop-common:jar:3.1.1-TDP-0.1.0-SNAPSHOT, org.apache.hadoop:hadoop-distcp:jar:3.1.1-TDP-0.1.0-SNAPSHOT, org.apache.hadoop:hadoop-annotations:jar:3.1.1-TDP-0.1.0-SNAPSHOT, org.apache.hadoop:hadoop-mapreduce-client-core:jar:3.1.1-TDP-0.1.0-SNAPSHOT, org.apache.hadoop:hadoop-hdfs:jar:3.1.1-TDP-0.1.0-SNAPSHOT: Failure to find org.apache.hadoop:hadoop-auth:jar:3.1.1-TDP-0.1.0-SNAPSHOT in https://repository.apache.org/content/repositories/releases/ was cached in the local repository, resolution will not be reattempted until the update interval of apache release has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :phoenix-hbase-compat-2.4.1
```

--- a/pom.xml
+++ b/pom.xml
@@ -84,7 +84,7 @@
     <!-- This is used by the release script only -->
     <hbase.profile.list>2.1 2.2 2.3 2.4.0 2.4</hbase.profile.list>
     <!-- The default hbase versions to build with (override with hbase.version) -->
-    <hbase-2.1.runtime.version>2.1.10-TDP-0.1.0-SNAPSHOT</hbase-2.1.runtime.version>
+    <hbase-2.1.runtime.version>2.1.10-1.0</hbase-2.1.runtime.version>
     <hbase-2.2.runtime.version>2.2.7</hbase-2.2.runtime.version>
     <hbase-2.3.runtime.version>2.3.6</hbase-2.3.runtime.version>
     <hbase-2.4.0.runtime.version>2.4.0</hbase-2.4.0.runtime.version>
@@ -1605,7 +1605,7 @@
         <hbase.profile>2.1</hbase.profile>
         <hbase.compat.version>2.1.6</hbase.compat.version>
         <hbase.version>${hbase-2.1.runtime.version}</hbase.version>
-        <hadoop.version>3.1.1-TDP-0.1.0-SNAPSHOT</hadoop.version>
+        <hadoop.version>3.1.1-0.0</hadoop.version>
         <tephra.hbase.compat.version>2.1</tephra.hbase.compat.version>
       </properties>
     </profile>

--- a/tdp/README.md
+++ b/tdp/README.md
@@ -1,6 +1,6 @@
 # TDP Phoenix Notes
 
-The version 5.1.3-TDP-0.1.0-SNAPSHOT of Apache Phoenix is based on the branch `5.1` tag of the Apache [repository](https://github.com/apache/phoenix/tree/5.1).
+The version 5.1.2-1.0 of Apache Phoenix is based on the branch `5.1` tag of the Apache [repository](https://github.com/apache/phoenix/tree/5.1).
 
 ## Making a release
 
@@ -8,7 +8,7 @@ The version 5.1.3-TDP-0.1.0-SNAPSHOT of Apache Phoenix is based on the branch `5
 mvn clean install -DskipTests -Dhbase.profile=2.1
 ```
 
-This command generates `phoenix-hbase-2.1-5.1.3-TDP-0.1.0-SNAPSHOT-bin.tar.gz` file in the `phoenix-assembly/target` directory.
+This command generates `phoenix-hbase-2.1-5.1.2-1.0-bin.tar.gz` file in the `phoenix-assembly/target` directory.
 
 ## To run tests
 
@@ -17,7 +17,7 @@ This command generates `phoenix-hbase-2.1-5.1.3-TDP-0.1.0-SNAPSHOT-bin.tar.gz` f
 mvn test -Dhbase.profile=2.1 -DPhoenixPatchProcess -Dskip.code-coverage
 ```
 
-- -Dhbase.profile=2.1, builds phoenix with hbase 2.1.10-TDP-0.1.0-SNAPSHOT
+- -Dhbase.profile=2.1, builds phoenix with hbase 2.1.10-1.0 and hadoop 3.1.1-0.0
 - -DPhoenixPatchProcess, disables the build of the shaded artifacts (not necessary for tests)
 - -Dskip.code-coverage, self explanatory
 
@@ -27,6 +27,6 @@ mvn test -Dhbase.profile=2.1 -DPhoenixPatchProcess -Dskip.code-coverage
 mvn verify -Dhbase.profile=2.1 -DPhoenixPatchProcess -Dskip.code-coverage
 ```
 
-- -Dhbase.profile=2.1, builds phoenix with hbase 2.1.10-TDP-0.1.0-SNAPSHOT
+- -Dhbase.profile=2.1, builds phoenix with hbase 2.1.10-1.0 and hadoop 3.1.1-0.0
 - -DPhoenixPatchProcess, disables the build of the shaded artifacts (not necessary for tests)
 - -Dskip.code-coverage, self explanatory

